server/models/Expense.js

```diff
@@ -141,6 +142,32 @@ export default function (Sequelize, DataTypes) {
       deletedAt: {
         type: DataTypes.DATE,
       },
+
+      tags: {
+        type: DataTypes.ARRAY(DataTypes.STRING),
+        set(tags) {
+          if (tags) {
```

Comment:
Can we add a comment explaining why we're doing that?

Reply:
I know see that the code was duplicated from somewhere else, can that be extracted in the `tags` lib?